### PR TITLE
change to the `.format()` function

### PR DESCRIPTION
**Pull request by MarkusHackspacher**

### CODE REVIEW

**Comment by LinuxDonald** [comment]
@MasterofJOKers can you please review it?

**Comment by janexx** [comment]
I am no programmer, but it seems that just the formatting is changed?

**Comment by LinuxDonald** [comment]
Yes but i dont have any plan of it. And when Joker say its okay it will get merged. He is our activ coder.

**Comment by MasterofJOKers** [comment]
Thank you for your contribution. Is this commit part of a larger project of yours?
Could you please integrate the other pull request into this one and change the commit message to be more specific? It currently reads, as if you are changing the whole project to use `.format()` instead of just one specific file.

Other than that, you are also not doing just that. You're changing the layout of comments. Please remove the parts changing comments from the commit. It looks like you're trying to force pep8 comments, but our coding style says: 

> Line length: We don't comply strictly with the 79 characters. Usually we try to stay around 90 columns per line.

Some of the changes you propose don't look good. This could be helped with by using up to 90 characters. I'll comment on an example next.

File: horizons/util/yamlcachestorage.py [hunk 1]
```diff
@@ -122,4 +132,5 @@ def __contains__(self, item):
 		return item in self._data
 
 	def __str__(self):
-		return "YamlCacheStorage('%s', %d items)" % (self._filename, len(self._data))
+		return "YamlCacheStorage('{0!s}', {1!d} items)".format(self._filename,
+			len(self._data))
```
Comment (MasterofJOKers) [inline horizons/util/yamlcachestorage.py]: Using 90 characters would probably help the readability of this change.

**Comment by MarkusHackspacher** [comment]
@MasterofJOKers I change my commit, is it now ok?

File: horizons/util/yamlcachestorage.py [hunk 2]
```diff
@@ -97,10 +99,12 @@ def sync(self):
...
 		except Exception as e:
 			# Ignore all exceptions because saving the cache on disk is not critical.
-			self.log.warning("Warning: Unable to save cache into %s: %s" % (self._filename, unicode(e)))
+			self.log.warning("Warning: Unable to save cache into {0!s}: {1!s}".
+				format(self._filename, unicode(e)))
 
 	def close(self):
-		"""Write the file to disk if possible and then invalidate the object in memory."""
+		"""Write the file to disk if possible and then invalidate
+		the object in memory."""
```
Comment (MasterofJOKers) [inline horizons/util/yamlcachestorage.py]: Did you miss that one?

**Comment by MarkusHackspacher** [comment]
@MasterofJOKers yes I miss, and make one commit with the other format() changes from the yamlcache.py file.

**Comment by MasterofJOKers** [comment]
Thank you. This looks good now. I'll squash the commits and merge them as "Use `.format()` for string formatting in yamlcache".

**Comment by MasterofJOKers** [comment]
Squashed and merged into master.

**Pull request closed without merging**